android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiWebChromeClient.java

```diff
@@ -395,7 +400,7 @@ public boolean interceptOnBackPressed()
 	 * Returns false if not and the system should do its default handling.
 	 */
 	@Override
-	@RequiresApi(21)
+	@RequiresApi(23)
```

Comment:
The [WebChromeClient.onShowFileChooser()](https://developer.android.com/reference/android/webkit/WebChromeClient#onShowFileChooser(android.webkit.WebView,%20android.webkit.ValueCallback%3Candroid.net.Uri[]%3E,%20android.webkit.WebChromeClient.FileChooserParams)) method is documented for API Level 21. Why are you changing it to 23? The only thing API Level 23 related within this method is the permission request, but for Android 5.x we still want to display the file chooser.